src/app/Http/Controllers/PostController.php

```diff
@@ -22,9 +24,9 @@ public function index()
      *
      * @return \Illuminate\Http\Response
      */
-    public function create()
-    {
-        //
+    public function create(Request $request)
+    {    //
+        
```

Comment:
mislim da bi bilo dobro da odradis https://laravel.com/docs/8.x/validation#quick-writing-the-validation-logic validaciju ovde na nivou requesta mozda? da ne bi morao svaki put da izvlacis na ovaj nacin vec samo iz requesta da cupas data